streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -356,27 +347,14 @@ public void shouldCloseStandbyTasksOnShutdown() {
         verify(activeTaskCreator);
     }
 
-    @Ignore
-    @Test
-    public void shouldUnassignChangelogPartitionsOnShutdown() {
-        throw new RuntimeException();
-//        restoreConsumer.unsubscribe();
```

Comment:
Added this check to shouldCloseActiveTasksOnShutdown (although now, it's a changelogReader expectation)